app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::CommentsController < Api::ApplicationController
+  def index
+    @comments = Post.find(params[:post_id]).comments
+    render json: @comments
+  end
+
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = Comment.create!(params.require(:comment).permit(:text)
+     .merge(author_id: @user.id, posts_id: @post.id))
```

Comment:
- [ ] I was unable to create a comment for a post due to an error. I think you may want to use `params[:user_id]` instead since there is no `current_user` in both the `controllers` and `api/controllers`.

  ```
  ...
  .merge(author_id: params[:user_id], posts_id: @post.id))
  ```

  ![image](https://user-images.githubusercontent.com/54848363/225889593-e0c26a0e-ed59-4c77-b813-45aff7aaa8f2.png)

Reply:
@DammyShittu 
Thanks for the feedback

Reply:
Fixed.